ab_tool/constants.py

```diff
@@ -5,4 +5,8 @@
 ADMINS = [ADMINISTRATOR, CONTENT_DEVELOPER, TEACHING_ASSISTANT, INSTRUCTOR]
 STAGE_URL_TAG = 'intervention_point_url_'
 DEPLOY_OPTION_TAG = 'is_canvas_page_for_track_'
-AS_TAB_TAG = 'open_as_tab_for_track_'
+
+NAME_CHAR_LIMIT = 256
```

Comment:
If models refer to these constants, changes to these constants will require a database migration. Should such behavior belong in settings?